src/lib.rs

```diff
@@ -1,3 +1,5 @@
+// Copyright 2019 Centrality Investments Limited
```

Comment:
Could u add the rest of the license
(example is here: https://github.com/cennznet/doughnut-rs/blob/167a442b141f554fdcd7e9a44b70bed6a60f2b52/src/lib.rs#L1-L13)